app/models/ability.rb

```diff
@@ -0,0 +1,14 @@
+# class Ability
+#   include CanCan::Ability
+
+#   def initialize(user)
+#     return unless user.present?
+
+#     if user.admin?
+#       can :manage, :all
+#     else
+#       can %i[read create], :all
+#       can %i[update destroy], Food, user: user
+#     end
+#   end
+# end
```

Comment:
- [ ] As required you are supposed to work on the authorization, so please implement it instead of comment out the code.

 To implement authorization, when a user signs up, you need to send a confirmation email as (a link) in the terminal before a user signing in, just to authorize a user.

Optional suggestion,
 `you can implement admin actions`

Reply:
Fixed